package.json

```diff
@@ -23,10 +23,10 @@
       "import": "./dist/js/uswds.min.js",
       "default": "./dist/js/uswds.min.js"
     },
-    "./src/js/components": "./src/components/index.js",
-    "./src/js/components/*": "./src/js/components/*.js",
-    "./src/fonts/*": "./src/fonts/*",
-    "./src/img/*": "./src/img/*",
+    "./src/js/components": "./packages/uswds-core/src/js/index.js",
+    "./src/js/components/*": "./packages/*.js",
```

Comment:
This is legacy and doesn't seem practical with the new structure. 

**Example**
```js
import accordion from "@uswds/uswds/src/js/components/usa-accordion/src/accordion";
```